proxy/proxyUtils.js

```diff
@@ -91,15 +91,15 @@ const handleSessionExpiry = (proxyRes, proxyResData, req, res, error, data) => {
   } else if(error || errorStatus.includes(proxyRes.statusCode)) {
     edata['message'] = `${req.originalUrl} failed`;
     edata.level = "ERROR";
-    logger.info({message: `${req.originalUrl} failed`});
+    logger.info({ message: `${req.originalUrl} failed` });
     logMessage(edata, req);
     const resCode = errorResponse(req, res,proxyRes, error);
     // logging the Error events
     telemetryHelper.logTelemetryErrorEvent(req, data, proxyResData, proxyRes, resCode);
     return resCode;
   } else {
     edata['message'] = `${req.originalUrl} successfull`;
-    logger.info({message: `${req.originalUrl} successfull`});
+    logger.info({ message: `${req.originalUrl} successfull` });
     logMessage(edata, req);
     auditEventObject(req, proxyResData);
```

Comment:
verify if audit event flag is enabled or not, if enable then trigger audit event.